src/pages/studyView/chartHeader/ChartHeader.tsx

```diff
@@ -689,7 +689,7 @@ export class ChartHeader extends React.Component<IChartHeaderProps, {}> {
                             >
                                 <button
                                     className={classnames(
-                                        'btn btn-xs btn-default',
+                                        'btn btn-xs btn-default deleteChart',
```

Comment:
If this is only for testing purpose, maybe we can use `data-test={'deleteChart'}`?